### PR TITLE
[Examples] Add word count and pagerank

### PR DESCRIPTION
**Pull request by Yuzhen11**

### CODE REVIEW

File: backend/python_connector.cpp [hunk 1]
```diff
@@ -80,7 +80,7 @@ void PythonConnector::start_python_proc(int lid, int gid, int num_workers) {
     std::string master_port = std::to_string(Context::get_config().get_master_port());
     std::string proc_id = std::to_string(Context::get_worker_info().get_process_id());
     extern_py_proc =
-        popen(("python -m pyhusky.backend.python_backend " + std::to_string(lid) + " " + std::to_string(gid) + " " +
+        popen(("python -m python.pyhusky.backend.python_backend " + std::to_string(lid) + " " + std::to_string(gid) + " " +
```
Comment (kygx-legend) [inline backend/python_connector.cpp]: No need to change this. By default, `$PYTHONPATH` in readme includes python dir.

File: backend/register.cpp [hunk 2]
```diff
@@ -17,6 +17,8 @@
 #include "backend/library/functional.hpp"
 #include "backend/library/linear_regression.hpp"
 #include "backend/library/logistic_regression.hpp"
+#include "backend/library/word.hpp"
```
Comment (kygx-legend) [inline backend/register.cpp]: wrong order?

File: python/pyhusky/backend/library/functional.py [hunk 3]
```diff
@@ -16,8 +16,8 @@
 
 import cloudpickle
 
-from pyhusky.backend.globalvar import GlobalVar, GlobalSocket, OperationParam, GlobalN2NSocket
-from pyhusky.common.serializers import Serializer
+from python.pyhusky.backend.globalvar import GlobalVar, GlobalSocket, OperationParam, GlobalN2NSocket
```
Comment (kygx-legend) [inline python/pyhusky/backend/library/functional.py]: ditto

**Review by kygx-legend — CHANGES_REQUESTED** [review]
Please follow the readme again.

**Comment by kygx-legend** [comment]
Ref #28

File: python/pyhusky/backend/library/functional.py (hunk 3 above)
Comment (Yuzhen11) [reply python/pyhusky/backend/library/functional.py]: I don't want to use the installed one. I met some problem during re-installation. The package cannot update. So instead I directly use the raw python file in the folder.

Comment (kygx-legend) [reply python/pyhusky/backend/library/functional.py]: Certainly yes for development. Then you don't need to use the installed method. A very simple way is to set env `$PYTHONPATH` like `$ export PYTHONPATH=$PYHUSKY_ROOT/python` and this is compatible with `import pyhusky.*`. Adding `python.pyhusky` is too ugly.

Comment (kygx-legend) [reply python/pyhusky/backend/library/functional.py]: Sorry for that I don't remember to add this into readme.

**Review by kygx-legend — APPROVED** [review]

**Pull request merged**